src/component/framework/form-group.js

```diff
@@ -31,7 +37,7 @@ export class FormGroup {
    */
   @computedFrom('element')
```

Comment:
sidenote: does that work? i don't remember. or does it need to be @computedFrom('element.key', 'element.label')?